strongbox-storage/strongbox-storage-api/src/main/java/org/carlspring/strongbox/providers/io/RepositoryFileAttributeType.java

```diff
@@ -21,8 +21,6 @@
     
     ARTIFACT_PATH("artifactPath"),
 
-    RESOURCE_URL("resourceUrl"),
```

Comment:
FYI: I've removed this attribute since it's no longer needed. The URLs are now generated through  `StrongboxUriComponentsBuilder` since it provides support for reverse proxy and can properly pick-up the `host/domain` from the incoming request.